UI/TicketUI/TicketUI.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Logic;
+using Model;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace UI.TicketUI
+{
+    public partial class TicketUI : Form
+    {
+        TicketService ticketService;
```

Comment:
You could put this locally in InitListView().

Reply:
Fixed